Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM golang:1.13 AS build-env
```

Comment:
Can we update this to match the version in the go.mod please?

https://github.com/FairwindsOps/pluto/blob/81c8d3298a4415075750d4ccdb1580b15c9b972e/go.mod#L3